lightning-invoice/src/payment.rs

```diff
@@ -1729,66 +1589,6 @@ mod tests {
 		});
 	}
 
-	#[test]
```

Comment:
we no longer populate inflight path information based on events, so there is probably no need to have tests for this any longer.